maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,101 @@
+        elif message.role == MessageRole.SYSTEM:
+            return GenerationRequestMessage(
+                role="system",
+                content=parse_message_content(message.blocks)
+            )
+        elif message.role == MessageRole.TOOL:
+            return GenerationRequestMessage(
+                role="tool",
+                content=parse_message_content(message.blocks)
+            )
+        else:
+            scribe().warning(f"Unsupported message role: {message.role}")
```

Ruff (0.12.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)